forge-gui/res/cardsfolder/upcoming/magar_of_the_magic_strings.txt

```diff
@@ -0,0 +1,11 @@
+Name:Magar of the Magic Strings
```

Comment:
Need to fix it in SetStateEffect or something:
> There are no cards in this set that would turn a face-down instant or sorcery card on the battlefield face up, but some older cards can try to do this. If something tries to turn a face-down instant or sorcery card on the battlefield face up, reveal that card to show all players it’s an instant or sorcery card. The permanent remains on the battlefield face down. Abilities that trigger when a permanent turns face up won’t trigger, because even though you revealed the card, it never turned face up. 

Currently they just disappear forever 😱